### DISCUSSION TITLE
QR Code Asset Tag Information

### DISCUSSION THREAD

**Discussion by dkleckner498** [amidaware/tacticalrmm#1034] [Ideas]
Not sure if this has been a request or idea yet.  Under the Asset Tab of the Server/Workstation (Asset) create a new Tab with the QR Code that links to a quick page of the asset and have each asset create a quick information page that within the console pops up and gives general information about the asset such as the Client it Belongs to, the location, the information under Summary Tab.  So that you can create an asset label and attach to the asset and scan with a phone or device to pop up on the web the asset information after authenticating to the RMM Portal.  Built in Asset Management.

**Comment by dinger1986** [comment]
I think there's a snipeit integration being worked on

**Reply by dkleckner498** [reply]
That would be awesome

**Comment by plao** [comment]
👍
